app/policies/permit_application_policy.rb

```diff
@@ -32,6 +32,10 @@ def submit?
     update?
   end
 
+  def generate_missing_pdfs?
+    true
```

Comment:
shouldnt there be some check that the user has the right to see this PDF

Reply:
@bkmaibach you are right. My initial assumption was all authenticated user should be able to generate this permit application once submitted. But there should be a check that submitters own the permit application and reviewers belong to the same jurisdiction. Good catch